src/sidebar/services/annotations-exporter.ts

```diff
@@ -25,18 +25,20 @@ export class AnnotationsExporter {
   /**
    * @param now - Test seam
    */
-  buildExportContent(now = new Date()): ExportContent {
+  buildExportContent(
+    annotations: Annotation[],
+    now = new Date()
```

Comment:
I presume this is a test seam?

Reply:
Yep. Just pushed the missing `istanbul ignore next`